root/etc/cont-init.d/20-drupal-files-dirs

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/with-contenv bash
+
+for DIR in $(ls -d ${DOCROOT}/sites/*/files); do
+  s6-echo -- Fixing ownership and permissions of $DIR
+  chown apache:apache $DIR
```

Comment:
So, do you think that not permitted warning is fine for most folks to see and wont confuse things?